package.json

```diff
@@ -24,37 +24,43 @@
     "postinstall": "./node_modules/.bin/opencollective-postinstall || exit 0"
   },
   "dependencies": {
-    "@babel/runtime": "^7.0.0-beta.38",
+    "@babel/runtime": "^7.0.0-beta.44",
```

Comment:
Same here...  I prefer to have a clean PR with the minimal packages changes related to this PR (without unnecessary meteor or npm packages upgrades). We can do a newer PR with meteor/npm upgrades, so if we have any problem with some commit in, for instance, production, we can identify correctly when/why something stopped to work, or a new bug arise, etc...